Templates/MonoGame.Templates.CSharp/content/MonoGame.2D.StartKit/___SafeGameName___.Android/___SafeGameName___.Android.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0-android</TargetFramework>
+    <SupportedOSPlatformVersion>23</SupportedOSPlatformVersion>
+    <OutputType>Exe</OutputType>
+    <ApplicationId>com.companyname.___SafeGameName___</ApplicationId>
+    <ApplicationVersion>1</ApplicationVersion>
+    <ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <AndroidResource Include="Resources\Drawable\Splash.png" />
+    <AndroidResource Include="Resources\Values\Styles.xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <MonoGameContentReference Include="..\___SafeGameName___.Core\Content\___SafeGameName___.mgcb">
```

Comment:
Should this be just `..\___SafeGameName___.Core\Content\Content.mgcb`?

The reason is it effects the value used in 
`Content.RootDirectory = "Content";`

Or at least it used to.

Reply:
@dellis1972 it doesn't appear to, everything runs fine, and maybe shouldn't because it's talking about the root directory and not the content file within the root directory. Anyone else know??